### ISSUE TITLE
How to use it in Python + Windows

### ISSUE THREAD

**Issue by josephernest** [google/leveldb#528]
What is the "official" implementation of LevelDB in Python 2.7? Is it [`plyvel`](https://plyvel.readthedocs.io/en/latest/)? If so, it doesn't work out of the box with Windows 7:

    pip install plyvel

generates compilation errors.

A simple easy-to-install would be cool for the popularity of LevelDB.

**Comment by pwnall** [comment]
We do not maintain / support any Python implementation of LevelDB at the moment. I think your best options for getting help quickly are asking the authors of the project you're using, or asking on StackOverflow.

**Comment by AdamRosenthal** [comment]
Is this still true?

**Comment by pwnall** [comment]
Yes

**Comment by pwnall** [comment]
leveldb.org is not affiliated with this project. The name is rather unfortunate.

**Comment by josephernest** [comment]
Could someone give the big picture of the various leveldb Github, leveldb.org, plyvel, leveldb-py, etc. project?
@pwnall @adamr2 @google-admin 

**TL;DR: How to use Level DB with Python 3.7 as of today, on Windows?**

**Comment by pwnall** [comment]
Sorry, I'm going to lock this conversation. As mentioned above, this repository is focused on the LevelDB implementation, which is in C++.
